README.md

```diff
@@ -49,8 +52,6 @@ To enable and test:
 
   1. Enable using real AWS by supplying the use_real_aws parameter to the contrib/ansible/deploy-devel.yaml playbook.
 		* `ansible-playbook contrib/ansible/deploy-devel.yaml -e "use_real_aws=true"`
-  1. `cp ./contrib/examples/cluster.yaml ./contrib/examples/mycluster.yaml`
-  1. Edit mycluster.yaml and change the name to your username. This will allow you to find objects created in the AWS account to clean up.
-  1. `kubectl create -f ./contrib/examples/mycluster.yaml`
+  1. Create a Cluster. Prefer a name for the Cluster that is prefaced with your username. This makes it easier to associate to you resources created in the AWS, making them easier to find and clean up. By default, if you are using the contrib/examples/create-cluster.sh script, the cluster will be prefaced with your username.
```

Comment:
You should add the command to create the cluster in a separate code-formatted block so it's easier to find when scanning through the  doc (as you had the kubectl command before)

Reply:
man, sorry please ignore. I see you have it above.